### ISSUE TITLE
Disable the GitHub Wiki

### ISSUE THREAD

**Issue by pradyunsg** [pythonindia/junction#632]
I don't think we should be using the Wiki for PyCon India specific workflow items; which is what most of the Wiki is for.

**Comment by ananyo2012** [comment]
Yeah we need to port it somewhere else. Probably https://github.com/pythonindia/pyconindia-handbook/wiki

**Comment by palnabarun** [comment]
+1 to moving it to PyCon India handbook.

**Comment by palnabarun** [comment]
### Tasks
- [ ] Move over stuff from junction wiki to handbook
- [ ] Disable wiki

**Comment by palnabarun** [comment]
PS: Anyone looking at this issue from the devsprint, this is not part of the sprint as it requires write credentials to the organizations.
